server/api/documents.js

```diff
@@ -388,6 +392,7 @@ router.post('documents.info', auth({ required: false }), async ctx => {
     document = await Document.findByPk(
       id,
       user ? { userId: user.id } : undefined
+      // XXX: this seems crazy to me, like you can only find the doc if you created it?
```

Comment:
@tommoor slightly confused by this line, I am under the impression that the option scopes the find to match for userId? This seems like only the author would find the document, but I'm certainly misunderstanding something

Reply:
It's the second argument, which means it's passed into the `options` and allows us to preload the users memberships, here:
https://github.com/outline/outline/blob/master/server/models/Document.js#L213-L217